vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -115,20 +115,25 @@ function Get-FilesToCacheOnVHD {
             "https://acs-mirror.azureedge.net/kubernetes/v1.17.9-azs/windowszip/v1.17.9-azs-1int.zip",
             "https://acs-mirror.azureedge.net/kubernetes/v1.17.11-azs/windowszip/v1.17.11-azs-1int.zip",
             "https://acs-mirror.azureedge.net/kubernetes/v1.16.10-hotfix.20200817/windowszip/v1.16.10-hotfix.20200817-1int.zip",
+            "https://acs-mirror.azureedge.net/kubernetes/v1.16.10-hotfix.20200824/windowszip/v1.16.10-hotfix.20200824-1int.zip",
```

Comment:
I think that we can remove this one since 20200917 is added

Reply:
I think that we do not need to add any new package for 1.16.10 since AKS does not support it

Reply:
Thanks for your review, Abel. I removed the two hotfixes for 1.16.10.